contracts/migrations/2_deploy_contracts.js

```diff
@@ -1,8 +1,14 @@
-var ListingsRegistry = artifacts.require("./ListingsRegistry.sol");
-var Listing = artifacts.require("./Listing.sol");
-var UserRegistry = artifacts.require("./UserRegistry.sol");
-var PurchaseLibrary = artifacts.require("./PurchaseLibrary.sol");
-var OriginIdentity = artifacts.require("./OriginIdentity.sol");
+var ClaimHolder = artifacts.require("./ClaimHolder.sol")
```

Comment:
use `const` instead of `var`. Related to https://github.com/OriginProtocol/origin-js/issues/192